ui/pages/SummaryData/components/Hpo.jsx

```diff
@@ -96,10 +121,23 @@ class Hpo extends React.PureComponent {
         ))}
         <Divider />
         {terms.length > 0 && (
-          <Header size="small">
-            <Header.Content>{`${families.size} Families, ${data.length} Individuals: `}</Header.Content>
-            <HorizontalSpacer width={10} />
-            <ButtonLink as={NavLink} disabled={!searchHref} target="_blank" to={searchHref}>Variant Search</ButtonLink>
+          <Header size="medium">
+            <Header.Content>{`${numFamilies} Families, ${data.length} Individuals`}</Header.Content>
+            <Header.Subheader>
+              {genomeSearchPaths.map(([genomeVersion, searchPath]) => (
+                <span key={genomeVersion}>
+                  {`${GENOME_VERSION_LOOKUP[genomeVersion]}: `}
```

Comment:
It may need a space -- `&nbsp;` at the end of the line.

Reply:
nope, the interpolated string with the space at the end actually renders the space

Reply:
Hmm, good to know.